packages/consumption/src/consumption/CoreErrors.ts

```diff
@@ -2,10 +2,10 @@ import { ApplicationError } from "@js-soft/ts-utils";
 import { CoreError, CoreId } from "@nmshd/transport";
 
 class Attributes {
-    public genericValidationError(error: any) {
+    public successorIsNotAValidAttribute(error: any) {
         return new CoreError(
-            "error.consumption.attributes.genericValidationError",
-            "Validation failed during creation of object.",
+            "error.consumption.attributes.successorIsNotAValidAttribute",
+            "Validation failed during creation of successor.",
             error,
             undefined,
             error instanceof Error ? error : undefined
```

Comment:
This error differs from the others in that it takes a parameter. E.g. we don't further explain _why_ an attribute is not a valid repository attribute or where a source attribute's content differs from that of its attribute copy. Any reason to break this pattern here?

This is more of a minor nitpick than anything else. I'll let you decide on whether you want to keep the detail or not. I just found it surprising reading through the errors in that your code suggests that the `successorIsNotAValidAttribute` error is in some way "special".

Reply:
This error if for the serval errors - I don't know if there is a good way to subdivide those

Reply:
No need to subdivide. Removing the parameter altogether is what I meant.

```ts
    public successorIsNotAValidAttribute() {
        return new CoreError(
            "error.consumption.attributes.successorIsNotAValidAttribute",
            "Validation failed during creation of successor."
        );
    }
```

Reply:
I want to keep the information on what exactly failed - I assume there is some valuable info :)